install_config/cluster_metrics.adoc

```diff
@@ -376,6 +376,37 @@ The deployer template parameter options and their defaults are listed in the
 default *_metrics-deployer.yaml_* file. If required, you can override these
 values when creating the Metrics Deployer.
 
+.Template Parameters
+[options="header"]
+|===
+
+|Parameter |Description
+
+|`*METRIC_DURATION*`
+|The number of days metrics should be stored.
+
+|`*CASSANDRA_PV_SIZE*`
+|The persistent volume size for each of the Cassandra nodes.
+
+|`*USE_PERSISTENT_STORAGE*`
+|Set to *true* for persistent storage, set to *false* to use non-persistent storage.
```

Comment:
The comma above should be a semicolon or period, or divided with an "or": "...persistent storage, or set to..."